### ISSUE TITLE
Corneal pannus

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6798]
**Preferred term label:**
Corneal pannus

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:106210
PMID: 16543198    5/10

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is either synonymous or closely related to https://hpo.jax.org/app/browse/term/HP:0011496 (Corneal neovascularization).

@psergouniotis -- can you advise? 
This is the definition from (https://webeye.ophth.uiowa.edu/eyeforum/atlas/pages/pannus.html)

Corneal pannus is subepithelial fibrovascular tissue ingrowth from the limbus onto the cornea. 

But it seems that you always get both new blood vessels and some fibrous tissue, so actually these are the same?

**Comment by psergouniotis** [comment]
this is not entirely straightforward. I think it would make sense to have those as synonyms (with "corneal neovascularisation" as the primary term, Alternatively, "corneal pannus" can be a subclass of "corneal neovasculatisation". 

I slightly prefer the former option.

**Comment by pnrobinson** [comment]
Thanks, @psergouniotis , I added corneal pannus as a narrow synonym. If there is ever a concrete need we can make it a subclass.
